src/dq_suite/common.py

```diff
@@ -71,19 +72,56 @@ def __getitem__(self, key) -> str | RulesList | None:
         raise KeyError(key)
 
 
+@dataclass()
+class DatasetDict:
+    """
+    Groups a list of Rule-objects together with the name of the table
```

Comment:
Deze docstring is nog een kopie van de `RulesDict` dataclass :) even omschrijven naar deze dataclass.